packages/api/e2e/amino.spec.ts

```diff
@@ -264,6 +265,24 @@ xdescribe('RegenApi with tendermint connection - Amino Tests', () => {
 
       await runAminoTest(msgClient, TEST_MSG_CANCEL);
     });
+    it('should sign and broadcast MsgBridge using legacy amino sign mode', async () => {
+      const { msgClient } = await connect();
+      const TEST_BRIDGE_BATCH_DENOM = 'C02-001-20200101-20210101-001';
+
+      const TEST_MSG_BRIDGE = MsgBridge.fromPartial({
+        owner: TEST_ADDRESS,
+        recipient: makeEthContract(),
+        target: 'polygon',
+        credits: [
+          {
+            batchDenom: TEST_BRIDGE_BATCH_DENOM,
+            amount: MIN_CREDIT_AMOUNT,
+          },
+        ],
+      });
+
+      await runAminoTest(msgClient, TEST_MSG_BRIDGE);
+    });
   });
```

Comment:
I have tested this against redwood locally, including all negative tests I could think of, but we will need to implement #59 to run it in CI.

https://redwood.regen.aneka.io/txs/C70A43E236BAAB1B2DD2D352A03CBB72D8DF307EB7F03D61905315B26324C738